server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -77,6 +77,18 @@ public class ShuffleServerConf extends RssBaseConf {
           .defaultValue(10 * 1000L)
           .withDescription("Heartbeat interval to Coordinator (ms)");
 
+  public static final ConfigOption<Long> SERVER_DIRECT_MEMORY_USAGE_REPORT_DELAY =
+      ConfigOptions.key("rss.server.direct.memory.report.delay")
+          .longType()
+          .defaultValue(10 * 1000L)
+          .withDescription("rss direct memory usage report initial delay (ms)");
+
+  public static final ConfigOption<Long> SERVER_DIRECT_MEMORY_USAGE_REPORT_INTERVAL =
+      ConfigOptions.key("rss.server.direct.memory.usage.report.interval")
```

Comment:
How about `rss.server.netty.direct-memory-tracker.metrics.update-interval-ms`

Reply:
Ok, I corrented it.

Reply:
Sorry for my comment incorrectness. Follows @jerqi suggestion here: https://github.com/apache/incubator-uniffle/pull/1370#discussion_r1425586987 

Let's use the camel style config key. 

How about `rss.server.netty.directMemoryTracker.memoryUsage.initialFetchDelayMs` and `rss.server.netty.directMemoryTracker.memoryUsage.updateMetricsIntervalMs` ? cc @jerqi

Reply:
Ok， I corrected it .